binary/asn1/schema.asn

```diff
@@ -33,7 +33,7 @@ Configuration-Structure-Schema DEFINITIONS AUTOMATIC TAGS ::= BEGIN
       ...
    }
    Configuration ::= SEQUENCE {
-      harts SEQUENCE OF Hart OPTIONAL,
+      harts SEQUENCE (SIZE(1..64)) OF Hart OPTIONAL,
```

Comment:
I don't think we can limit the size like that. E.g. Esperanto has a core with more than 1000 harts already.

Reply:
This is just the experiment of using SIZE syntax to see if we would get the reduced size. The entire schema should still have room to tailor for the concise size.